config/routes.rb

```diff
@@ -1,5 +1,11 @@
 Rails.application.routes.draw do
 
+  get 'static_pages/home'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.